source/_components/alarm_control_panel.paradox.markdown

```diff
@@ -0,0 +1,47 @@
+---
+layout: page
+title: "Paradox Alarm Control Panel"
+description: "Instructions how to integrate Paradox alarms into Home Assistant."
+date: 2016-10-07 12:17
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: paradox.png
+ha_category: Alarm
+ha_release: 0.30
+ha_iot_class: "Local Push"
+---
+
+The `paradox` alarm control platform is a sub module of the [Paradox hub](/components/paradox/) which provides integration with Paradox alarms utilizing the PRT3 module.
```

Comment:
HA devices are refered to as Components. In this case it's even more important since the actual device is also referred to as a module.